homeassistant/components/modern_forms/__init__.py

```diff
@@ -0,0 +1,180 @@
+"""The Modern Forms integration."""
+from __future__ import annotations
+
+import asyncio
+from datetime import timedelta
+import logging
+
+from aiomodernforms import (
+    ModernFormsConnectionError,
+    ModernFormsDevice,
+    ModernFormsError,
+)
+from aiomodernforms.models import Device
```

Comment:
You could consider doing like this to make it clearer which `Device` this is:
```suggestion
from aiomodernforms.models import Device as ModernFormsDevice
```

Reply:
Yeah fair enough